### PR TITLE
Use modules

### PR DESCRIPTION
**Pull request by iBel251**
- Create 3 modules for each functionality.
- Import all modules to the index.js file.
- Import luxon for date and time functionality.
- Refactor all classes and functions to arrow functions.
- Create a readme file.

### CODE REVIEW

File: modules/addbook.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const collection = document.querySelector('.collection');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBtn = document.querySelector('#add-btn');
+let id = 1 || JSON.parse(localStorage.getItem('maxID'));
+
+class BookObject {
```
Comment (EricMbouwe) [inline modules/addbook.js:7]: - [ ] You should also implement the` remove book` function as this is a project requirement. Let's fix that

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @iBel251,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addbook.js (hunk 1 above)
Comment (iBel251) [reply modules/addbook.js:7]: But the remove book functionality is there, it works.

Comment (iBel251) [reply modules/addbook.js:7]: https://github.com/iBel251/awesome-books-modules/blob/f165efe45702bd3d7d40a1de81a4fd18feb0669e/modules/addbook.js#L37

**Review by mikethreels — APPROVED** [review]
Hi Imran, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addbook.js (hunk 1 above)
Comment (EricMbouwe) [reply modules/addbook.js:7]: Okay my bad. I missed that, just request for another CR it will be approved!

**Pull request merged**